pydm/display.py

```diff
@@ -119,6 +119,53 @@ def load_ui_file(uifile, macros=None):
     return d
 
 
+def load_adl_file(filename, args=None, macros=None):
```

Comment:
To match the other on the other methods...
```suggestion
def load_adl_file(filename, macros=None, args=None):
```

Reply:
Fixed after rebase in a4982f63